admin_guide/sdn_troubleshooting.adoc

```diff
@@ -0,0 +1,547 @@
+= Troubleshooting OpenShift SDN
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+As described in the
+link:/architecture/additional_concepts/sdn.html[SDN documentation]
```

Comment:
Should be:

`link:../architecture/additional_concepts/sdn.html[SDN documentation]`